go.mod

```diff
@@ -19,9 +20,9 @@ require (
 	github.com/golang/protobuf v1.5.3 // indirect
 	github.com/kr/text v0.2.0 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	golang.org/x/net v0.34.0 // indirect
-	golang.org/x/sys v0.29.0 // indirect
-	golang.org/x/text v0.21.0 // indirect
+	golang.org/x/net v0.36.0 // indirect
```

Comment:
golang.org/x/net v0.36.0 requires a go version >= 1.23.0. So should we bump go version in this library, wdyt @ycombinator @swiatekm? 🙂

Reply:
ok you already took care of this @ycombinator , so would it make sense also to remove toolchain now?! 🙂

Reply:
https://github.com/elastic/elastic-agent-client/pull/136